cardano-testnet/src/Testnet/Components/Configuration.hs

```diff
@@ -179,6 +176,15 @@ createSPOGenesisAndFiles nPoolNodes nDelReps maxSupply asbe@(AnyShelleyBasedEra
     , "--out-dir", tempAbsPath
     ]
 
+  -- Overwrite the genesis files created by create-testnet-data with the files
+  -- specified by the user (if any)
+  case genesisOrigin of
```

Comment:
This decision should not be here. The point is if we are calling `createSPOGenesisAndFiles` we are already creating default files because the user has not provided a node configuration file.